network-api/networkapi/buyersguide/templates/category_page.html

```diff
@@ -15,11 +15,11 @@ <h1 class="h3-heading">{{ category.name }}</h1>
       {% endif %}
     </div>
 
-    <div class="categories-container">
+    <div class="d-flex flex-wrap row">
```

Comment:
We can clean this and SCSS code further by utilizing Bootstrap's `.row`, `.col-*`,  and `.justify-content-center`.

The trick is for medium screen size and up we can put all our content in a container that's `col-md-8` wide and make sure this container is positioned at center. It's kinda hard to explain this in words so I've included my code suggestion below

```html
{% block guts %}
  <div class="category-listing container mt-4">
    <div class="row justify-content-center">
      <div class="col-12 col-md-8">
        <div class="mb-5">
          <h1 class="h3-heading">{{ category.name }}</h1>
          {% if category.description != None %}
          <p class="body-large mt-4">{{ category.description }}</p>
          {% endif %}
        </div>
        <div class="row">
          {% for product in products %}
          <div class="category-item-container col-6 ...
```

This way you can get rid of the following custom styling we have for `.category-listing` (you might need to keep/adjust padding)
```scss
.category-listing {
  background: #fff;
  padding-top: 2em;
  width: 100%;
  margin: 0 auto;
  max-width: 545px;

  @media screen and (min-width: $bp-md) {
    max-width: 800px;
  }
```

Note that you can also remove `text-center` and `text-left` classes from `category_page.html` since by default text is aligned to left.

Reply:
Really good suggestions.